packages/fxa-content-server/app/scripts/templates/get_flow.mustache

```diff
@@ -0,0 +1,9 @@
+<div id="main-content" class="card">
+  <header>
+    <h1>{{#t}}One moment{{/t}}</h1>
+  </header>
+
+  <section>
+    {{#t}}You should be redirected shortly...{{/t}}
+  </section>
+</div>
```

Comment:
Perhaps rename https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/app/scripts/templates/subscriptions_redirect.mustache and share between the two views for consistency.